_posts/itohosagie/2021-11-29-itohosagie-meetup-write.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: post
+author: itohosagie
+title: Meet-up Write-up: Workshop on Spatial Data in SQL Server
```

Comment:
To ensure all of this title is understood as a string, it is safest to enclose it in quotes `"`. I think some of the characters in this title (e.g., `:`) will cause trouble, otherwise.